01/app.js

```diff
@@ -1 +1,10 @@
-console.log('DOM');
\ No newline at end of file
+console.log('DOM');
+
+const commentsItem = document.querySelectorAll('.comments__item.comments__item--newest');
+console.log('commentsItem.length', commentsItem.length);
+
+if(commentsItem) {
```

Comment:
Pamiętaj, że jak masz `querySelectorAll` to zwracany element jest zawsze prawdą. Pusta tablica to też prawda. 
Jak już chcesz sprawdzić czy coś zostało wyszukane to musisz zrobić `commentsItem.length > 0` lub samo `commentsItem.length` bo 0 to wartość fałszywa.

Reply:
dziekuje za cenna uwage ;)